app/bundles/UserBundle/EventListener/RouteSubscriber.php

```diff
@@ -11,13 +11,28 @@
 
 namespace Mautic\UserBundle\EventListener;
 
-use Mautic\CoreBundle\EventListener\CommonSubscriber;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\GetResponseEvent;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\HttpKernel\KernelEvents;
 
-class RouteSubscriber extends CommonSubscriber
+class RouteSubscriber implements EventSubscriberInterface
 {
+    /**
+     * Core params.
+     *
+     * @var array
+     */
+    private $params;
+
+    /**
+     * @param array $params
+     */
+    public function __construct(array $params)
```

Comment:
Why are we injecting the parameters here instead of the `CoreParametersHelper` service?

Reply:
https://github.com/mautic/mautic/pull/8148#discussion_r355351359